### ISSUE TITLE
PostgreSQL connection: RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range.

### ISSUE THREAD

**Issue by v01t** [rickbergfalk/sqlpad#659]
On creation or selection of postgres database connection, container exit with following error:
```
{"level":20,"time":"2020-06-01T23:17:21.623Z","pid":1,"hostname":"b7680c18634c","name":"sqlpad-app","msg":"Executing (default): SELECT `id`, `name`, `data`, `expiry_date` AS `expiryDate`, `created_at` AS `createdAt` FROM `cache` AS `Cache` WHERE `Cache`.`id` = 'schemacache:57760320-26f3-55ff-a100-ac49532c593f';"}
internal/buffer.js:62
    throw new ERR_OUT_OF_RANGE('value', range, value);
    ^

RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received 9_007_199_254_740_992
    at checkInt (internal/buffer.js:62:11)
    at writeU_Int32BE (internal/buffer.js:792:3)
    at Buffer.writeUInt32BE (internal/buffer.js:805:10)
    at Object.execute (/usr/app/node_modules/pg-protocol/dist/serializer.js:125:10)
    at Connection.execute (/usr/app/node_modules/pg/lib/connection.js:162:24)
    at Cursor._getRows (/usr/app/node_modules/pg-cursor/index.js:170:19)
    at Cursor._shiftQueue (/usr/app/node_modules/pg-cursor/index.js:80:19)
    at Cursor.handleRowDescription (/usr/app/node_modules/pg-cursor/index.js:96:8)
    at Connection.<anonymous> (/usr/app/node_modules/pg/lib/client.js:324:22)
    at Connection.emit (events.js:323:22)
    at /usr/app/node_modules/pg/lib/connection.js:109:10
    at Parser.parse (/usr/app/node_modules/pg-protocol/dist/parser.js:42:17)
    at Socket.<anonymous> (/usr/app/node_modules/pg-protocol/dist/index.js:8:42)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:209:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  code: 'ERR_OUT_OF_RANGE'
}
```
configuration:
```
{
  "allowCsvDownload": true,
  "cookieName": "sqlpad.sid",
  "cookieSecret": "xxx",
  "dbPath": "/data/db",
  "disableAuth": true,
  "disableAuthDefaultRole": "admin",
  "disableUserpassAuth": true,
  "editorWordWrap": true,
  "ip": "0.0.0.0",
  "port": 80,
  "publicUrl": "https://sqlpad.com",
  "queryResultMaxRows": 50000,
  "sessionMinutes": 60,
  "systemdSocket": false,
  "timeoutSeconds": 300,
  "queryHistoryRetentionTimeInDays": 90,
  "appLogLevel": "debug",
  "webLogLevel": "warn",
  "seedDataPath": "/data/seed",
  "authProxyEnabled": false,
  "authProxyAutoSignUp": false,
  "connections": {
     "test": {
      "name": "test",
      "driver": "postgres",
      "host": "test",
      "database": "postgres",
      "username": "sqlpad",
      "password": "sqlpad",
      "multiStatementTransactionEnabled": true,
      "idleTimeoutSeconds": 86400
    }
  }
}
```
compose file:

```
version: '3.3'

configs:
  sqlpad:
    external: true

services:
  app:
    image: sqlpad/sqlpad
    environment:
      SQLPAD_CONFIG: /etc/sqlpad/config.json
    configs:
      - source: sqlpad
        target: /etc/sqlpad/config.json
    networks:
      - test
    volumes:
      - data:/data

  test:
    image: postgres
    environment:
      POSTGRES_USER: sqlpad
      POSTGRES_PASSWORD: sqlpad
    networks:
      - test

volumes:
  data:

networks:
  test:
    external: true
```

**Comment by rickbergfalk** [comment]
Hi @v01t - This is happening prior to trying to run any queries? 

I'll see if I can reproduce

**Comment by v01t** [comment]
Yes, it happens as soon as connection selected in the UI (if it was previously defined in the configuration) or created in the UI.

Thanks

**Comment by rickbergfalk** [comment]
I think I maybe got this fixed, but not sure as I wasn't able to reproduce. 

I noticed in the error message `It must be >= 0 and <= 4294967295. Received 9_007_199_254_740_992`. That latter number is JavaScript's MAX_SAFE_INTEGER, and it looks like it is too large for 32 bit int?

I've changed the implementation to fetch up to 1 million rows for schema, which should be enough. Would probably have issues if schemas were actually that large anyways. 

Want to give `latest`/`master` a shot and let me know if that cleared things up?
